CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
         - Add aerial maps option to Bing and OSM maps.
         - Select matches for both filter category and group. #3110
         - Add an extra zoom level to most map types. #3130
+        - Display search tips when location search returns no results. #3180
```

Comment:
Probably should be in UK section further down now.

Reply:
Good idea!